const notFound = (_req, res) => {
  res.status(404).send({ error: 'Recurso n√£o encontrado' });
};

module.exports = notFound;
